uspekhi-gerontologii.csl

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" default-locale="ru">
+  <info>
+    <title>Uspekhi-gerontologii(numeric, sorted alphabetically, Russian)</title>
+    <id>http://www.zotero.org/styles/uspekhi-gerontologii</id>
+    <link href="http://www.zotero.org/styles/uspekhi-gerontologii" rel="self"/>
+    <link href="http://www.gersociety.ru/information/uspexi/" rel="documentation"/>
```

Comment:
Please add a "template" link pointing to the GOST style. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link

Should be `<link href="http://www.zotero.org/styles/gost-r-7-0-5-2008-numeric-alphabetical" rel="template"/>`, if you used that particular style as a starting point.